integrations/kotlin-support/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
```

Comment:
Do not leave empty lines within copyright

Reply:
removed